includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -105,6 +118,43 @@ public function register_routes() {
 						'type'              => 'string',
 						'validate_callback' => [ $this, 'validate_statement_descriptor' ],
 					],
+					'account_business_name'             => [
+						'description' => __( 'The customer-facing business name.', 'woocommerce-payments' ),
+						'type'        => 'string',
+					],
+					'account_business_url'              => [
+						'description' => __( 'The customer-facing business url.', 'woocommerce-payments' ),
```

Comment:
It'd be nice to pick up more accurate descriptions from https://stripe.com/docs/api/accounts/update#update_account-business_profile and https://stripe.com/docs/api/accounts/update#update_account-settings-branding. For descriptions that are too long in Stripe docs, we can shorten them smartly.